Packs/Gatewatcher/ReleaseNotes/0_1_0.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### New: Gcenter v102
+- This is the Gatewatcher integration for getting started.
```

Comment:
Because this is a brand new pack there is no need to provide release notes. 
Please delete the file (it will solve the build's error as well).

Reply:
done